pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions_test.go

```diff
@@ -478,6 +480,129 @@ func TestAccessModeConflicts(t *testing.T) {
 	}
 }
 
+func Test_isSchedulableAfterPersistentVolumeClaimChange(t *testing.T) {
```

Comment:
Can we clean up test cases based on the current implementation? I believe we can simplify each test case and PVCs that we're using.

e.g., I think we don't need a consideration to `ReadWriteOncePod` etc.

Reply:
Cleaned up.(I cleaned it up once before, but I missed some content)